news/2024/2024-10-30-osu-world-cup-2024-qualifiers-recap.md

```diff
@@ -0,0 +1,144 @@
+
+Doing all of that while managing a full-custom qualifier pool, which means we have limited control of how difficult maps are in relation to each other, and how much the niche we want them to fill is actually expressed to begin with, means that it's all a pretty difficult endeavor. I think we ended up doing a decent job of it, though.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/7944724" style="background-image: url('/wiki/shared/news/2024-07-12-osu-catch-world-cup-2024-concludes/avatar-0x84f.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/7944724">0x84f</a></p>
+
+How does the map pooling team determine difficulty of Qualifiers map pools, and has there been an exponential increase in difficulty over the past few years?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/6701656" style="background-image: url('/wiki/shared/news/2024-10-30-osu-world-cup-2024-qualifiers-recap/avatar-chiv.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/6701656">chiv</a></p>
+
+It's common practice in most tournaments to base the qualifier difficulty off of the mid-range stages, and OWC is no exception. The Qualifiers this year are around 7.3*, which is parallel to our current Quarterfinals difficulty. The mid-range scope allows for a general assessment of a team's skill cap and consistency capabilities. It's important to recognize that while seeing your favourite players battle it out on extremely difficult maps may be incredible, we can't forget about consistency being the most vital skill set in tournament gameplay.
+
+As you've probably recognized, OWC has seen a drastic upward shift in difficulty over the last few years. Due to multiple factors like technological development, skill set refinement and overall player growth, the tournament as a whole has gotten much harder (OWC'24 RO32 being comparable to OWC'20 QF). Qualifiers has followed suit alongside the rest of the mappools. As players continue to grow and hone their craft, we'll do our best to continue to match the expectations.
+
+</div>
+
+---
+
+For the next month and a half you will have the chance to witness top tier osu! tournament gameplay, so be sure to check out the wiki go familiarize yourselves with the mappools, schedules, and all other details [on the osu! wiki](/wiki/Tournaments/OWC/2024), get involved with the community live on the official [osu! Twitch channel](https://www.twitch.tv/osulive), and support your favourite players and teams.
```

Comment:
For the next month and a half, get ready to immerse yourself in top-tier osu! tournament action! Check out the [osu! World Cup wiki page](/wiki/Tournaments/OWC/2024) to download and try the mappools yourself, see the schedules for when your favourite teams are playing, and all of the other essential details. Join us on stream live this weekend for the Round of 32 at [osulive](https://www.twitch.tv/osulive) or [osulive_2](https://www.twitch.tv/osulive_2) to support your favorite players and teams.

Join us next week for more exciting updates and news!

—0x84f, ChillierPear, Dada